src/containers/Users/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { apiService } from '../../services';
+
+import { Users } from './Users';
+
+export const UsersContainer: React.FC = () => {
+
+  const [users, setUsers] = React.useState([]);
+
+  React.useEffect(() => {
+    apiService('https://jsonplaceholder.typicode.com/users')
```

Comment:
BAAAAD
you should pass the URL to the service in the appropriate file
remove URL